cmd/terramate/cli/cli.go

```diff
@@ -206,9 +206,7 @@ func newCLI(
 
 	cwd, err := os.Getwd()
 	if err != nil {
-		log.Fatal().
-			Err(err).
-			Msg("Getwd() failed")
+		return nil, fmt.Errorf("Getwd() failed: %v", err)
```

Comment:
but this should be zero log and executed here to show correct line in coffee . how to do this?

Reply:
hmm there are some alternatives. Some libraries embed automatically on error instances this information, so we could keep using the error and have line of code info. Another alternative is use log.Panic(), it will produce the message you want + a panic stack trace.

Reply:
makes not much sense to me.. it's a fatal thing and not a broken thing.. ;) 

as this is initialization there should not be any cleanup to do anyway.. why not just exit?

Reply:
Just returning the error will also just exit with an error status code. The fastest way right now would be to log with an error and still return the error, calling Exit here (or fatal) will not work well with how the cli tests are written today, we can change the tests to work with code calling Exit, but would need some time for this (change the test approach, probably not much hopefully).

Reply:
but this is not using zerolog then.. which is now the logging used here..

Reply:
Yeah it would not use it for Fatal/exiting. I'm going to work on changing the tests so calling exit/log.Fatal would be OK anywhere on the code, hopefully it won't take much time.

Reply:
btw tests are still failing.. ;)

Reply:
we can change it to error but how to return then?

Reply:
yeah there is something else related to os.Getwd/Chdir (which was changed from original code on main) that is breaking the tests, Im taking a look into it. What this was "fixing" is the test aborting all of a sudden and then there was no error message at all or even the name of the test that failed etc.

Reply:
maybe the initialization should go to main.. not sure why we need to test commandline parsing.. and chdir..

Reply:
It is supposed to an end to end test, like ensuring that running:

```
terramate init
```

Works, in an e2e manner. We tried to do this extracting all the logic to cli.Run (so main doesn't to anything), but then some side effect operations can cause problems since just calling a function cli.Run is not as isolated as running a separate process. Overall doing e2e tests with a separate process will increase isolation of the tests, so an aborting terramate can't kill the entire test process.